### DISCUSSION TITLE
Driver not working on Big Sur 11.6.4

### DISCUSSION THREAD

**Discussion by NavinSrivastava** [chris1111/Wireless-USB-Big-Sur-Adapter#185] [Q&A] [answered]
Hi Chris
After updating to 11.6.4 drivers are not working, specifically after rebooting.
Kernel extensions are not getting added either in SIP enabled or disabled
Please help

**Comment by chris1111** [comment]
Uninstall then reinstall

**Comment by NavinSrivastava** [comment]
I have done that already, it does not work after a reboot even with SIP
disabled

**Reply by NavinSrivastava** [reply]
I do not get the notification for kernel extension and reboot when I have uninstalled and installed again in 11.6.4

**Comment by chris1111** [comment]
I have BS 11.6.4 and its work
what is the model of your mac? 
what is the exact model of your adapter

**Reply by NavinSrivastava** [reply]
I have a Macbook Pro (Retina 13 inch, late 2013)
2.4 dual core intel i5 8 GB 1600 Mhz DDR3 RAM
For uninstallation, I have used uninstall option in the installer
I had also followed steps to enable SIP executing  consent.command 

My wifi adapter is RTL8811AU Wireless LAN 802.11 ac USB 2.0 Network Adapter

**Reply by NavinSrivastava** [reply]
Hi Chris 
Any suggestions? Is there anyway I can remove the permissions given to Kernel extension and retry installation. This works when it is installed, but after reboot, USB network adapter is not detected at all.

**Comment by chris1111** [comment]
I have a Macbook Pro (Retina 13 inch, late 2013) and its work without issue

**Comment by chris1111** [comment]
Anyway this is an issue [not a discussion](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/33)

This will be deleted, its not respect the rules.

**Comment by chris1111** [comment, marked as answer]
This will be deleted, its not respect the rules.
